docs/en/source/quickstart/install.md

```diff
@@ -1,3 +1,86 @@
-# Installation
+# Install TinyMS
 
-TODO
+## User install TinyMS
+
+### Pypi
+
+For users owning a clean environment, recommend using [pypi](https://pypi.org/) to install TinyMS after satisfy the following prerequisites. For setting up python environment, [Anaconda](https://www.anaconda.com/products/individual#Downloads) is a good choice.
+
+Prerequisites  
+
+- Ubuntu: `18.04`
+- Python: `3.7.5`
+
+Chinese Domestic users recommend running the following code setting the mirror site of pypi to solve the downloading issue
```

Comment:
For China based users it is recommended to run ..... to help with faster download